en/components/calendar.md

```diff
@@ -143,31 +143,41 @@ If everything went well, we should now have a calendar with customized dates dis
 </div>
 
 #### Events
-Let's build on top of that sample a bit. We will require the user to enter a date range that does not exceed 5 days. We need to change the `selection` mode of the calendar to "range" and prompt the user to correct the selection, if the range is not valid. In order to do this, we will use the [`onSelection`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#onselection) event:
+Let's explore the events emitted by the calendar:
+- [`onSelection`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#onselection) - emitted when selecting date(s) in the calendar.
+- [`onViewDateChanged`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#onsviewdatechanged) - emitted after the active view is changed - for example after the user has clicked on the `month` or `year` section in the header
+- [`onActiveViewChanged`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#onactiveviewchanged) - emitted every time when the current month/year is changed - for example after navigating to the `next` or `previous` month.
```

Comment:
the examples for the events onViewDateChanged and onActiveViewChanged are misplaced